src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -148,6 +148,7 @@ private static JsonObject locationSummary(Location location) {
     write(locationSummary, "name", location.getName());
     write(locationSummary, "libraryName", location.getLibraryName());
     write(locationSummary, "code", location.getCode());
+    write(locationSummary, "effectiveLocationPrimaryServicePointName", location.getPrimaryServicePoint().getName());
```

Comment:
Could getPrimaryServicePoint() sometimes return null? If so, you should probably have a null check.